### DISCUSSION TITLE
Which file do i choose to load a osk to lazer?

### DISCUSSION THREAD

**Discussion by NukeThief** [ppy/osu#20987] [Q&A]
I play osu! only using lazer. And i have no idea which exe do i choose when loading a skin. Do i have to use the ingame import files feature or am i doing something wrong?

**Comment by NotGumballer91** [comment]
Yes. Or you can just drag your skin file (.osk). That works too

**Reply by NukeThief** [reply]
so theres no open file thing yet?

**Reply by Doksha** [reply]
> so theres no open file thing yet?

there is that feature, but its not properly implemented (it doesnt work when the game is running.) basically the only way to import skins is to click and drag, then hit alt tab to change the window to lazer
